### PR TITLE
Improvment - Addition of custom model via extra-action

### PR DESCRIPTION
**Pull request by Craym-**
I modified two places in the document: Around line 352 so that a two
news extra_actions elements are available : "responseClass"  as well as
"model" in order to allow the user to define custom model for his custom
endpoints.
The other modification is on line 563+ to add the custom model onto the
modellist.

This is my first commit / pull request ever. So let me know if I should do this differently. Thanks!

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -115,6 +115,24 @@ Top level keys and meaning in the ``extra_actions`` dictionary:
   and include a ``{id}`` parameter in the uri and parameter list.
 - ``summary``: Description of this endpoint.
 - ``fields``: **Optional** Dictionary of parameters this endpoint accepts.
+- ``responseClass``: **Optional** Should correspond to the id of the corresponding model
```
Comment (johnraz) [inline docs/index.rst]: I would reword this to ` Should match the id of the corresponding model`

File: docs/index.rst [hunk 2]
```diff
@@ -115,6 +115,24 @@ Top level keys and meaning in the ``extra_actions`` dictionary:
...
+- ``model``: **Optional** Dictionary of parameters describing the model. ``type`` and ``description`` are the two required elements.
+Example ::
+
+	'responseClass': 'video_managment',
```
Comment (johnraz) [inline docs/index.rst]: `management`

**Comment by johnraz** [comment]
Awesome ! Other than the 2 small changes above this LGTM. Merci !

**Comment by Craym-** [comment]
Ok, thanks for the feedback! :D
So what should I do now? Just do the changes you suggested and submit them to my own repository?

**Comment by johnraz** [comment]
Yep, apply the changes and push to the same branch

**Comment by Craym-** [comment]
The fix has been commited. I added as well a small fix concertning the fields that were showing all 'optional' from not reading the model from tastypie properly.
I could easily do some more changes and merge as well the code given by other users. I definitely need to keep improving this. Let me know how it is suppose to work.
- Do I commit more changes now or do I wait for your to do the pull and then propose the next update for it?
- Or can I keep updating the files and add functionalities?

Thanks!

**Comment by johnraz** [comment]
I will merge this and you can open new pull requests for other features.
Try to keep 1 pull per feature.

Also, if you plan on working on this project, I would recommend that you go through the various open issues as there are some open discussions there and I'm mostly inactive on the project myself.

Cheers.
